@xen-orchestra/backups/_cleanVm.test.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
-/* eslint-env jest */
+const { beforeEach, afterEach, test, describe } = require('test')
+const assert = require('assert')
```

Comment:
Use strict comparison:

```suggestion
const assert = require('assert').strict
```

Reply:
Either use `assert.strict.equal` or `assert.strictEqual`, but using both is unnecessary (cf my previous comment).